src/utils/emlTemplate.j2

```diff
@@ -89,40 +92,65 @@
             </temporalCoverage>
         </coverage>
 
-    <maintenance>
-      <description>
-        <para/>
-      </description>
-      <maintenanceUpdateFrequency>asNeeded</maintenanceUpdateFrequency>
-    </maintenance>
 
+        <methods>
+            {% for method in methodList %}
+            <methodStep>
+                <description>
+                    <section>
+                        <title>{{ method.title.en }}</title>
```

Comment:
My text editor (VSCode) is complaining about <title> being here. do you have any links to info on this <title> tag?

Reply:
also possible my editor is using the wrong schema files, if this works in the IPT thats good enough

Reply:
I consulted a few different sources but ultimately based my structure on the following example: https://ediorg.github.io/data-package-best-practices/methods.html

Based on the idea that the description for a methodStep element is a section and a section has a title.

This seems to bear out in the EML schema detailed here, but I'm no expert in this either - the following validator doesn't complain about it: https://knb.ecoinformatics.org/emlparser/

It does have some opinions on roles being specified for associated party: 
`The content of element 'associatedParty' is not complete. One of '{userId, role}' is expected.`